### PR TITLE
Feature/redesign landing page

### PR DESCRIPTION
**Pull request by omorilewa**
**What does this PR do?**

This PR adds design changes.

**Description of Task to be completed?**

Design changes are added.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -26,17 +26,19 @@
...
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.5.2/animate.min.css">
-  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:400,800,700,600,300">
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet">
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.5.2/animate.min.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,800,700,600,300" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600,700" rel="stylesheet">
   <link href="https://fonts.googleapis.com/css?family=Frank+Ruhl+Libre|Playfair+Display" rel="stylesheet">
   <link href="https://fonts.googleapis.com/css?family=Patua+One" rel="stylesheet">
```
Comment (DavidLemayian) [inline public/index.html]: Are we using either of these 2 anymore?

File: src/components/Home.js [hunk 2]
```diff
@@ -12,9 +12,12 @@ export default class Home extends Component {
   }
   render() {
     return (
-      <div>
+      <div className="container page-width">
         <NavigationBar resetCalculator={this.resetCalculator.bind(this)} />
-        <div id="spacer" style={{"height": "150px"}}></div>
+        <div className="illustration">
+          <img src="/illustration.svg" alt="banner" className="img-responsive"/>
```
Comment (DavidLemayian) [inline src/components/Home.js]: All images should go into an `/img/` folder.

File: src/components/Navbar.js [hunk 3]
```diff
@@ -10,29 +9,28 @@ class NavigationBar extends Component {
...
-            <NavItem eventKey={3} href=" https://facebook.com/CodeForAfrica" target="_blank">
-              <i className="fa fa-facebook" aria-hidden="true"></i>
-            </NavItem>
-            <NavItem href="https://twitter.com/code4africa" target="_blank">
-              <i className="fa fa-twitter" aria-hidden="true"></i>
-            </NavItem>
-          </Nav>
-        </Navbar.Collapse>
-      </Navbar >
+              <h2 className="check-your-med-prices">#CheckYourMedPrices</h2></NavLink>
```
Comment (DavidLemayian) [inline src/components/Navbar.js]: Is this supposed to have a `.brand` class and not `h2` according to bootstrap docs?

File: src/components/Stories.js [hunk 4]
```diff
@@ -8,67 +8,182 @@ export default class Stories extends Component {
...
+                <div className="left">
+                  <img src="mitchell-hollander.png" className="img-responsive" alt="drug" />
+                </div>
+                <div className="recent-story">
+                  <h3 className="heading heading-wording">Crisis looms as drug prices rise by 200%</h3>
+                  <div className="row topMargin">
+                    <div className="rightborder">
+                      <img src="punch-ng-logo-white.png" width="47px" height="14px" alt="punch-logo" />
+                    </div>
+                    <div className="rightborder2 font-small ">
```
Comment (DavidLemayian) [inline src/components/Stories.js]: `rightborder2` looks like an odd class name.

File: src/components/Stories.js [hunk 5]
```diff
@@ -8,67 +8,182 @@ export default class Stories extends Component {
...
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <hr />
+        <div className="stories">
+          <div style={{"width": "100%"}}>
```
Comment (DavidLemayian) [inline src/components/Stories.js]: Necessary?

File: src/components/Stories.js [hunk 6]
```diff
@@ -8,67 +8,182 @@ export default class Stories extends Component {
...
+            <div className="row">
+              {data.map((news, index) => (
+                <div key={index} className="col-lg-4 col-md-8 col-xs-12 col-sm-10">
+                  <div key={index} className="hover-test">
+                    <div className="set-height2">
```
Comment (DavidLemayian) [inline src/components/Stories.js]: `.set-height2` seems like a weird class name.

File: public/index.html (hunk 1 above)
Comment (andela-mabdussalam) [inline public/index.html]: No.

File: src/components/Home.js (hunk 2 above)
Comment (andela-mabdussalam) [inline src/components/Home.js]: Done.

File: src/components/Navbar.js (hunk 3 above)
Comment (andela-mabdussalam) [inline src/components/Navbar.js]: Yes it is.

File: src/components/Stories.js (hunk 4 above)
Comment (andela-mabdussalam) [inline src/components/Stories.js]: I totally agree.

File: src/components/Stories.js (hunk 6 above)
Comment (andela-mabdussalam) [inline src/components/Stories.js]: Totally.

File: src/components/Stories.js (hunk 5 above)
Comment (andela-mabdussalam) [inline src/components/Stories.js]: No.

**Pull request closed without merging**